### ISSUE TITLE
Mask loss is 0.0 and overall loss doesnt converge && what is relevance of dt_num_classes ?

### ISSUE THREAD

**Issue by Raj-08** [openseg-group/openseg.pytorch#84]
Hi , 
I am training segfix model and my mask loss is 0.0 for most of iterations. Also the loss doesnt converge. 

Another question i have is what is relevance of dt_num_classes? is it related to the actual total number of classes?

**Comment by xiamibudayang** [comment]
hello, I am training segfix model too and I also meet the seem problem as the same as you. Have you find the answer now? 
Thanks

**Comment by Raj-08** [comment]
Yes , the problem lies in your creation of direction maps . Visualise them and see if it makes sense as per the direction maps shown in research paper.
